lumigator/python/mzai/backend/backend/services/datasets.py

```diff
@@ -94,6 +95,7 @@ def _raise_not_found(self, dataset_id: UUID) -> None:
         raise HTTPException(status.HTTP_404_NOT_FOUND, f"Dataset '{dataset_id}' not found.")
 
     def _raise_unhandled_exception(self, e: Exception) -> None:
+        traceback.print_exception(type(e), e, e.__traceback__)
```

Comment:
we can simplify this with `traceback.print_exc()`
https://docs.python.org/3/library/traceback.html#traceback.print_exc

Reply:
Thanks!!!!! ^_^